mainchain-follower/db-sync-follower/src/candidates/datum/d_param.rs

```diff
@@ -1,18 +1,16 @@
+use super::PlutusDataExtensions;
```

Comment:
Looking in retrospective, I think it wasn't very good move to have Datums aka PlutusData in db-sync-follower. Format of Plutus data used will be same regardless we read it from db-sync, Kupo, Ogmios or whatever else. In the data it is only a cbor of Plutus data.

Reply:
Separate PR IMO

Reply:
You're right, we should have a separate crate with smart contract plutus data schemas. I can move it in a separate PR.